src/player.cpp

```diff
@@ -11258,7 +11254,7 @@ bool player::can_study_recipe(const itype &book) const
         auto const r = elem.recipe;
         if( !knows_recipe( r ) &&
             ( !r->skill_used ||
-              get_skill_level( r->skill_used ) >= elem.skill_level ) ) {
+              ( get_skill_level( r->skill_used ) - elem.skill_level ) >= 4 ) ) {
```

Comment:
This results in you needing to exceed the requirements by 4 levels to memorize a recipe? Could be factored out as a constant and possibly reduced to 3?